### PR TITLE
Updated guide pages - wrap true, false, null [skip ci]

### PR DESCRIPTION
**Pull request by SilverFire**
Review is required

### CODE REVIEW

File: docs/guide-es/caching-data.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ El siguiente código muestra el típico patrón de uso para el almacenamiento en
 // intenta recuperar $data de la caché
 $data = $cache->get($key);
 
-if ($data === false) {
+if ($data === `false`) {
```
Comment (cebe) [inline docs/guide-es/caching-data.md]: should not be done inside of PHP code examples. I'm afraid the PR is useless this way :)

File: docs/guide-fr/db-query-builder.md [hunk 2]
```diff
@@ -186,7 +186,7 @@ $query->where([
 ]);
 ```
 
-Comme vous pouvez le voir, le constructeur de requêtes est assez intelligent pour manipuler correctement les valeurs qui sont soit nulles, soit des tableaux.
+Comme vous pouvez le voir, le constructeur de requêtes est assez intelligent pour manipuler correctement les valeurs qui sont soit `null`es, soit des tableaux.
```
Comment (cebe) [inline docs/guide-fr/db-query-builder.md]: this is wrong

**Review by cebe — CHANGES_REQUESTED** [review]
except one bug, looks good.

File: docs/guide/tutorial-core-validators.md [hunk 3]
```diff
@@ -162,8 +162,8 @@ or `1970-01-01` in the input field of a date picker.
 
 ```php
 [
-    // set "age" to be null if it is empty
-    ['age', 'default', 'value' => null],
+    // set "age" to be `null` if it is empty
+    ['age', 'default', 'value' => `null`],
```
Comment (Mak-Di) [inline docs/guide/tutorial-core-validators.md]: php code

['age', 'default', 'value' => null],

**Review by Mak-Di — CHANGES_REQUESTED** [review]

File: docs/guide/tutorial-core-validators.md (hunk 3 above)
Comment (SilverFire) [inline docs/guide/tutorial-core-validators.md]: Fixed

File: docs/guide-fr/db-query-builder.md (hunk 2 above)
Comment (SilverFire) [inline docs/guide-fr/db-query-builder.md]: Fixed

**Review by cebe — APPROVED** [review]

**Review by samdark — APPROVED** [review]

**Pull request merged**